src/commands/registries/azure/deployImageToAca.ts

```diff
@@ -38,27 +38,32 @@ export async function deployImageToAca(context: IActionContext, node?: RemoteTag
     }
 
     const commandOptions: DeployImageToAcaOptionsContract = {
-        imageName: node.fullTag,
+        image: node.fullTag,
     };
 
-    addImageTaggingTelemetry(context, commandOptions.imageName, '');
+    addImageTaggingTelemetry(context, commandOptions.image, '');
 
     const registry: RegistryTreeItemBase = node.parent.parent;
     if (registry instanceof AzureRegistryTreeItem) {
         // No additional work to do; ACA can handle this on its own
-    } else if (registry instanceof DockerHubNamespaceTreeItem || registry instanceof DockerV2RegistryTreeItemBase) {
-        const { auth } = await registry.getDockerCliCredentials() as { auth?: { username?: string, password?: string } };
+    } else {
+        // Any other V2 compliant registry

```

Comment:
This comment is a _little_ misleading since GitLab is only sorta kinda is V2 compliant. Enough so that the Docker CLI can pull images, but that's about it.